src/components/profile/style/profile.less

```diff
@@ -54,23 +54,27 @@
   .ga-profile-grid-y {
     stroke: black;
     opacity: 0.8;
-    
+
     line {
       stroke-width: 0.02em;
     }
   }
- 
+
   .ga-profile-area {
     fill: #f00;
     fill-opacity: 0.5;
   }
-  
+
   .ga-profile-legend,
   .ga-profile-label {
     font-weight: bold;
     text-shadow: 1px 1px lightgrey;
   }
 
+  text {
+    cursor: default;
```

Comment:
I thought it made more sense with the tooltip. Not sure people want to copy paste that. But I am open to suggestions.